[2025-08-19T09:24:20.154+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-19T09:24:20.178+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_audit_dag.data_pull_task manual__2025-08-19T09:24:15.353741+00:00 [queued]>
[2025-08-19T09:24:20.186+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_audit_dag.data_pull_task manual__2025-08-19T09:24:15.353741+00:00 [queued]>
[2025-08-19T09:24:20.188+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-08-19T09:24:20.205+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): data_pull_task> on 2025-08-19 09:24:15.353741+00:00
[2025-08-19T09:24:20.213+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=121) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-08-19T09:24:20.215+0000] {standard_task_runner.py:63} INFO - Started process 125 to run task
[2025-08-19T09:24:20.214+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'data_audit_dag', 'data_pull_task', 'manual__2025-08-19T09:24:15.353741+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/Airflow Coding Assessment 2/data_audit_dag.py', '--cfg-path', '/tmp/tmp1l37sqvo']
[2025-08-19T09:24:20.217+0000] {standard_task_runner.py:91} INFO - Job 24: Subtask data_pull_task
[2025-08-19T09:24:20.276+0000] {task_command.py:426} INFO - Running <TaskInstance: data_audit_dag.data_pull_task manual__2025-08-19T09:24:15.353741+00:00 [running]> on host 6cacbbe28f9c
[2025-08-19T09:24:20.363+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='youremail@example.com' AIRFLOW_CTX_DAG_OWNER='Anbu Dharshini' AIRFLOW_CTX_DAG_ID='data_audit_dag' AIRFLOW_CTX_TASK_ID='data_pull_task' AIRFLOW_CTX_EXECUTION_DATE='2025-08-19T09:24:15.353741+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-19T09:24:15.353741+00:00'
[2025-08-19T09:24:20.365+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-19T09:24:20.367+0000] {logging_mixin.py:188} INFO - [Pull] Pulled Data: {'user_id': 101, 'balance': 884, 'timestamp': '2025-08-19 09:24:20.366959'}
[2025-08-19T09:24:20.368+0000] {python.py:237} INFO - Done. Returned value was: {'user_id': 101, 'balance': 884, 'timestamp': '2025-08-19 09:24:20.366959'}
[2025-08-19T09:24:20.368+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-19T09:24:20.394+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=data_audit_dag, task_id=data_pull_task, run_id=manual__2025-08-19T09:24:15.353741+00:00, execution_date=20250819T092415, start_date=20250819T092420, end_date=20250819T092420
[2025-08-19T09:24:20.431+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-08-19T09:24:20.459+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-08-19T09:24:20.463+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
